### PR TITLE
Feature/basic features

### PR DESCRIPTION
**Pull request by miladkdz**

### CODE REVIEW

File: demo/incubator-breadcrumb-basic-demos.html [hunk 1]
```diff
@@ -10,6 +10,9 @@ <h3>Demo sample</h3>
     <vaadin-demo-snippet id="incubator-breadcrumb-demo-sample">
       <template preserve-content>
         <incubator-breadcrumb>
+          <incubator-breadcrumb-step separator=">" href="#" shift>Home</incubator-breadcrumb-step>
```
Comment (tonygoldcrest) [inline demo/incubator-breadcrumb-basic-demos.html]: image
image

**Review by tonygoldcrest — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
-[&lt;incubator-breadcrumb&gt;](https://vaadin.com/components/incubator-breadcrumb) is a Web Component providing an easy way to ask the user to confirm a choice, part of the [Vaadin components](https://vaadin.com/components).
+[&lt;incubator-breadcrumbs&gt;](https://vaadin.com/components/incubator-breadcrumbs) is a Web Component providing an easy way to display breadcrumb on web pages.
 
 ```html
-  <incubator-breadcrumb header="Unsaved changes" confirm-text="Save" on-confirm="save"
-    cancel on-cancel="cancel" reject reject-text="Discard" on-reject="discard">
-    Do you want to save or discard your changes before navigating away?
-  </incubator-breadcrumb>
+  <incubator-breadcrumbs>
+    <incubator-breadcrumb separator=">" href="#">Home</incubator-breadcrumb>
```
Comment (tonygoldcrest) [inline README.md]: separator=">" should be removed

File: src/incubator-breadcrumb.html [hunk 3]
```diff
@@ -9,17 +9,31 @@
...
+    <style include="lumo-typography">
+      :host(:last-of-type) [part="separator"],
+      :host([last-step]) [part="separator"] {
+        display: none;
+      }
     </style>
 
-    <h1>Hello world</h1>
+    <span part="link">
+      <template is="dom-if" if="{{href}}">
```
Comment (tonygoldcrest) [inline src/incubator-breadcrumb.html]: {{}} -> [[]]

File: src/incubator-breadcrumb.html [hunk 4]
```diff
@@ -9,17 +9,31 @@
...
+        <a href="[[href]]"><slot></slot></a>
+      </template>
+      <template is="dom-if" if="{{href === ''}}">
```
Comment (tonygoldcrest) [inline src/incubator-breadcrumb.html]: {{}} -> [[]]

File: src/incubator-breadcrumbs.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        static get is() {
+          return 'incubator-breadcrumbs';
+        }
+
+        static get properties() {
+          return {
+
+            steps: {
+              type: Array,
+              value: []
```
Comment (tonygoldcrest) [inline src/incubator-breadcrumbs.html]: [] -> () => []

File: theme/lumo/incubator-breadcrumb-styles.html [hunk 6]
```diff
@@ -1,9 +1,22 @@
...
+      :host {
+        font-family: var(--lumo-font-family);
+      }
+
+      [part="separator"] {
+        margin: var(--lumo-space-wide-xs);
+      }
+
+      [part="separator"]::after {
+        content: '>';
```
Comment (tonygoldcrest) [inline theme/lumo/incubator-breadcrumb-styles.html]: This should not be separated from the component I guess

File: vaadin-directory-description.md [hunk 7]
```diff
@@ -1,17 +1,15 @@
...
 
 ## Example Usage
 
 ```html
-  <incubator-breadcrumb header="Unsaved changes" confirm-text="Save" on-confirm="save"
-    cancel on-cancel="cancel" reject reject-text="Discard" on-reject="discard">
-    Do you want to save or discard your changes before navigating away?
-  </incubator-breadcrumb>
+  <incubator-breadcrumbs>
+    <incubator-breadcrumb separator=">" href="#">Home</incubator-breadcrumb>
```
Comment (tonygoldcrest) [inline vaadin-directory-description.md]: separator=">"

**Review by tonygoldcrest — CHANGES_REQUESTED** [review]

File: theme/lumo/incubator-breadcrumb-styles.html (hunk 6 above)
Comment (miladkdz) [reply theme/lumo/incubator-breadcrumb-styles.html]: It's better that it come from theme. Something like a vaadin-icon for example. But for now I would settle for '>' only.

Comment (tonygoldcrest) [reply theme/lumo/incubator-breadcrumb-styles.html]: @miladkdz, you should be able to see something when you import the component from the src folder.

**Review by tonygoldcrest — CHANGES_REQUESTED** [review]

**Pull request merged**